Web/Cookie.hs

```diff
@@ -12,6 +12,8 @@ module Web.Cookie
     , setCookieDomain
     , setCookieHttpOnly
     , setCookieSecure
+    , setCookieSameSite
+    , SameSiteOption(..)
```

Comment:
If you aren't opposed, I'd rather switch to smart constructors in case additional SameSiteOption values become valid in the future. I can make that tweak myself if you're OK with it, or you can. What I would picture is:

* Only expose the `SameSiteOption` type, not its constructors
* Provide `sameSiteLax :: SameSiteOption` and `sameSiteStrict :: SameSiteOption` exports